pydantic/json.py

```diff
@@ -105,8 +105,8 @@ def custom_pydantic_encoder(type_encoders: Dict[Any, Callable[[Type[Any]], Any]]
 
 def timedelta_isoformat(td: datetime.timedelta) -> str:
     """
-    ISO 8601 encoding for timedeltas.
+    ISO 8601 encoding for time deltas.
```

Comment:
As the function accepts a [Python timedelta object](https://docs.python.org/3/library/datetime.html#timedelta-objects)(not  multiple), I would suggest changing it to `ISO 8601 encoding for Python timedelta object.`

```suggestion
    ISO 8601 encoding for Python timedelta object.
```

Reply:
even it was wrong before what you are suggesting is much beter documentation....  i will ammend my comit to change this